libraries/networking/src/DomainHandler.h

```diff
@@ -287,6 +292,8 @@ private slots:
     HifiSockAddr _iceServerSockAddr;
     NetworkPeer _icePeer;
     bool _isConnected { false };
+    bool _haveAskedConnectWithoutAvatarEntities { false };
+    bool _canConnectWithoutAvatarEntities { false };
```

Comment:
do these get reset when you switch domains?  should they?

Reply:
Yes, they get reset in `DomainHandler::hardReset()`.